lib/fileio/translators/translatorMesh.cpp

```diff
@@ -13,39 +13,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+// Modifications copyright (C) 2020 Autodesk
+//
+
 #include "translatorMesh.h"
 
-#include "../utils/meshUtil.h"
 #include "../../nodes/pointBasedDeformerNode.h"
-#include "../primReaderArgs.h"
-#include "../primReaderContext.h"
-#include "../utils/readUtil.h"
-#include "../utils/roundTripUtil.h"
 #include "../../nodes/stageNode.h"
-#include "translatorGprim.h"
-#include "translatorMaterial.h"
-#include "translatorUtil.h"
 #include "../../utils/util.h"
+#include "../utils/meshUtil.h"
+#include "../utils/readUtil.h"
```

Comment:
On my TODO to make the final changes we voted on and commit our coding guidelines to the repo. So yes, we should start using them now.

Reply:
@mattyjams Believe me I don't like the dots as much as you don't but I didn't want to spend time doing this in this PR since they are going to be cleanup in a serrate PR.